try{
} catch{
}

try {
}catch{
}

try {
}catch(error) {
}

if(a){
}else{
}

for(i = 0; i < n; i++) {}

for(x of y) {}
while(a)
    b();

async () => {
    for await(x of y) {}
};
